cabal-testsuite/PackageTests/NewConfigure/LocalConfigOverwrite/cabal.out

```diff
@@ -1,5 +1,5 @@
 # cabal v2-configure
-'cabal.project.local' file already exists. Now overwriting it.
+'cabal.project.local' already exists, copying it to 'cabal.project.local~'.
```

Comment:
`s/copying/backupin/`.

Though I don't remember what happens when `cabal.project.local~` already exists.

Reply:
It just gets overwritten it looks like

Reply:
Ok, let us fix that. Otherwise we could just overwrite `cabal.project.local` directly

Reply:
Fix by adding a message or by backing up cabal.project.local~ somewhere further?